<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centralWidget</class>
 <widget class="QWidget" name="centralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="Signal1">
           <property name="toolTip">
            <string extracomment="Represent signal of microphone1"/>
           </property>
           <property name="text">
            <string>Signal 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Signal 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Signal 3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Signal 4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>520</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSlider" name="verticalSlider">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>560</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Input">
        <attribute name="title">
         <string>Input</string>
        </attribute>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>130</y>
           <width>121</width>
           <height>30</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>48 ksps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24 ksps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12 ksps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 ksps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3 ksps</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Sampling rate</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>271</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Single event processing</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>271</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Continuous signal processing</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>251</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>TODO: Save triggered signals
 and load saved signals</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Windowing">
        <property name="toolTip">
         <string>Windowing</string>
        </property>
        <attribute name="title">
         <string>Windowing</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Window type</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>201</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maxVisibleItems">
          <number>10</number>
         </property>
         <item>
          <property name="text">
           <string>Rectangular</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triangular</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blackman</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hamming</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gauss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nuttall</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blackman-Harris</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blackman-Nuttall</string>
          </property>
         </item>
        </widget>
        <widget class="QGraphicsView" name="graphicsViewWindow">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>70</y>
           <width>150</width>
           <height>128</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>128</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>260</y>
           <width>201</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Signals after windowing</string>
         </property>
        </widget>
        <widget class="QGraphicsView" name="graphicsView_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>260</width>
           <height>128</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
        </widget>
        <widget class="QGraphicsView" name="graphicsView_6">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>290</y>
           <width>260</width>
           <height>128</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
        </widget>
        <widget class="QGraphicsView" name="graphicsView_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>440</y>
           <width>260</width>
           <height>128</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
        </widget>
        <widget class="QGraphicsView" name="graphicsView_8">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>440</y>
           <width>260</width>
           <height>128</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>128</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>160</y>
           <width>123</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Windowing</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Correlation">
        <attribute name="title">
         <string>Correlation</string>
        </attribute>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>400</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>GCC General Cross Corelation (unfiltered)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ML Maximum Likelihood</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PHAT weighting function</string>
          </property>
         </item>
        </widget>
        <widget class="QGraphicsView" name="graphicsView_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>512</width>
           <height>300</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>512</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>512</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>TDOA</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
